src/Modules/Unix/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1

```diff
@@ -34,14 +34,6 @@ HelpInfoURI = 'https://aka.ms/powershell71-help'
 PrivateData = @{
   PSData = @{
     ExperimentalFeatures = @(
```

Comment:
Maybe remove the `ExperimentalFeatures` key all together?

Reply:
I was thinking it might be easier to leave the placeholder if we add future ExperimentalFeatures, but if we want to remove this empty section, we should just remove all of PrivateData.  Thoughts?

Reply:
Yeah, I prefer to removing the whole `PrivateData` section to keep it clean.

Reply:
It is better to add the template to our docs for experimental features in the repo.

Reply:
Good point. It turns out our help doc doesn't mention how to declare experimental feature for a module:
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_experimental_features?view=powershell-7.2

@iSazonov Can you please open a doc issue?

Reply:
Done https://github.com/MicrosoftDocs/PowerShell-Docs/issues/8351